frontend/src/component/admin/network/NetworkTrafficUsage/NetworkTrafficUsage.tsx

```diff
@@ -42,30 +42,34 @@ const NetworkTrafficUsage: FC = () => {
 
     const { isOss } = useUiConfig();
 
-    const includedTraffic = useTrafficLimit();
+    const { trafficBundles } = useTrafficBundles();
+
+    const totalTraffic =
+        (trafficBundles.includedTraffic + trafficBundles.purchasedTraffic) *
+        1_000_000;
```

Comment:
There's a constant for traffic bundle default size somewhere I believe, might want to use that for consistency

Reply:
It was in the old hook I deleted. readded inside the useTrafficBundles() hook.